examples/sample_runtime/src/main.rs

```diff
@@ -309,23 +309,24 @@ fn main() {
 
         // Send a `StartService` request to the sample runtime.
         let instance_name = "instance".to_owned();
+
+        let start_service = StartService {
+            artifact: "255:sample_artifact".parse().unwrap(),
+            name: instance_name.clone(),
+            config: 10_u64.into_bytes(),
+        };
+
+        // `Height(0)` means that config will be applied in the next block.
```

Comment:
Nit: It could make sense to create a constructor for `ConfigPropose` that encapsulates this logic, e.g., `ConfigPropose::immediate(sequence_number)`.